databases/week1/lesson1_homework.sql

```diff
@@ -0,0 +1,56 @@
+-- 1. Find out how many tasks are in the task table
+SELECT count(id)
+FROM task;
+
+-- 2. Find out how many tasks in the task table do not have a valid due date
+SELECT count(id) AS tasks_with_invalid_due_date
+FROM task
+WHERE due_date IS NULL;
+
+-- 3. Find all the tasks that are marked as done
+SELECT *
+FROM task
+WHERE status_id = 3;
```

Comment:
in this case, you should use `JOIN` with the `status` table.  
We don't know what id `3` stand for